service/pod.go

```diff
@@ -0,0 +1,29 @@
+package service
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/model/pod"
+	"github.com/evergreen-ci/gimlet"
+)
+
+func (uis *UIServer) podPage(w http.ResponseWriter, r *http.Request) {
+	id := gimlet.GetVars(r)["pod_id"]
+
+	p, err := pod.FindOneByID(id)
+	if err != nil {
+		uis.LoggedError(w, r, http.StatusInternalServerError, err)
```

Comment:
can we wrap this error with "finding pod"?

Reply:
Done.